src/game_state/Level.h

```diff
@@ -0,0 +1,24 @@
+#ifndef LEVEL_H_
+#define LEVEL_H_
+
+#include <memory>
+#include <SFML/Audio.hpp>
+
+#include "GameObject.h"
+#include "Platform.h"
+
+class Level : public GameObject {
+ public:
+  Level(std::shared_ptr<sf::Music> music,
+        std::shared_ptr<std::vector<Platform>> level);
+  ~Level();
+
+  std::shared_ptr<sf::Music> getMusic() { return music; }
+  std::shared_ptr<std::vector<Platform>> getLevel() { return level; }
+
+ private:
+  std::shared_ptr<sf::Music> music;
+  std::shared_ptr<std::vector<Platform>> level;
```

Comment:
If all of the platforms are going here, then it would probably be best to store all other related GameObjects in other vectors here in place of having one vector of GameObjects in GameState